daemon/api_test.go

```diff
@@ -5024,3 +5024,24 @@ func (s *apiSuite) TestAliases(c *check.C) {
 	})
 
 }
+
+var _ = check.Suite(&postDebugSuite{})
+
+type postDebugSuite struct {
+	apiBaseSuite
+}
+
+func (s *postDebugSuite) TestPostDebugEnsureStateSoon(c *check.C) {
```

Comment:
This is not actually checking that the intended action was done, I think? Should be trivial to do that given the existing mocking around ensureStateSoon.